src/TypeSpec.Extension/Emitter.Csharp/package.json

```diff
@@ -50,18 +62,23 @@
     },
     "dependencies": {
         "@autorest/csharp": "3.0.0-beta.20230309.1",
-        "@azure-tools/typespec-azure-core": "0.35.0",
-        "@azure-tools/typespec-client-generator-core": "0.35.0",
-        "@typespec/compiler": "0.49.0",
-        "@typespec/eslint-config-typespec": "0.49.0",
-        "@typespec/eslint-plugin": "0.49.0",
-        "@typespec/http": "0.49.0",
-        "@typespec/library-linter": "0.49.0",
-        "@typespec/rest": "0.49.0",
-        "@typespec/versioning": "0.49.0",
         "fs-extra": "^10.1.0",
         "js-yaml": "^4.1.0",
         "json-serialize-refs": "0.1.0-0",
         "winston": "^3.8.2"
+    },
+    "peerDependencies": {
```

Comment:
Does typespec-csharp actually depend on all these?  typespec-python has a much smaller list:

```
    "peerDependencies": {
        "@azure-tools/typespec-azure-core": ">=0.36.0 <1.0.0",
        "@azure-tools/typespec-client-generator-core": ">=0.37.0-dev.3 <1.0.0",
        "@typespec/compiler": ">=0.50.0 <1.0.0",
        "@typespec/http": ">=0.50.0 <1.0.0",
        "@typespec/rest": ">=0.50.0 <1.0.0",
        "@typespec/versioning": ">=0.50.0 <1.0.0"
    },
```

https://github.com/Azure/autorest.python/blob/881659f763f4ee98d47fcf25a8082ca804d4a289/packages/typespec-python/package.json#L44

Reply:
I am just adding everything here because I don't want to have to deal with it any longer.